pay-api/src/pay_api/models/eft_refund.py

```diff
@@ -62,7 +62,6 @@ class EFTRefund(Audit):
     cas_supplier_site = db.Column(db.String(), nullable=True)
     comment = db.Column(db.String(), nullable=False)
     decline_reason = db.Column(db.String(), nullable=True)
-    created_by = db.Column("created_by", db.String(100), nullable=True)
```

Comment:
Already part of audit, removing because it is causing this to be blank when a row is created.